source/actors/template/coyotte_fall.tscn

```diff
@@ -1,12 +1,11 @@
 [gd_scene load_steps=2 format=2]
 
 [sub_resource type="GDScript" id=1]
-
 script/source = "extends Timer
 
 export (bool) var coyotte_fall = false
 
-func start():
+func time_start():
```

Comment:
What happened to the `Timer.start()` method? Was it crashing?

Just some context: this method was an override of the `Timer.start()`, so I wonder why the change to a custom method. If it wasn't crashing, could you undo this change, together with the [change on walk.gd line 40](https://github.com/pigdevstudio/platform_template/pull/48/files#diff-e4b347732e6427d9620993c408dea7d2R40), so the main interface of the the class is the same as its parent, Timer?

Reply:
I changed back from "time_start()" to "start()" and received the error that says: "error(5,1): Function signature doesn't match the parent. Parent signature is 'void start(float=default)'."

![platformer_template_timer_start_error](https://user-images.githubusercontent.com/51072029/68683643-0c9eff00-055f-11ea-88bd-7bfdd9101b8e.png)

Reply:
Ohh, then it is just a signature error, the Timer.start has a default parameter. Can you add this as

`func start(default_time = -1):`

please? This should fix it

Reply:
done